### PR TITLE
[NETBEANS-1074] module review javaee.api

### PR DESCRIPTION
**Pull request by rtaneja1**
Added license header in binaries-list, pointed to maven repository.
Changed to license CDDL 1.1 : https://javaee.github.io/glassfish/LICENSE

VerifyLibsandLicenses has open failure : enterprise/javaee.api/external/javax.annotation-api-1.2.jar and jaxrs-ri/ext/javax.annotation-api-1.2.jar in enterprise/websvc.restlib/external/jaxrs-ri-2.5.1.zip are identical
I believe this can be corrected by updated ignored-overlaps file after websvc.restlib module review is complete.

### CODE REVIEW

File: enterprise/javaee.api/external/javaee-api-6.0-license.txt [hunk 1]
```diff
@@ -1,270 +1,406 @@
 Name: JavaEE API 6.0
 Version: 6.0
-License: CDDL-GPL-2-CP
-OSR: Sun
+License: CDDL-1.1
 Description: JavaEE API 6.0
-Origin: Oracle (http://download.java.net/maven/2/javax/javaee-api/6.0/)
+OSR: GlassFish Community
```
[the remark this answers is not in the document]
Comment (rtaneja1) [reply enterprise/javaee.api/external/javaee-api-6.0-license.txt]: Done

File: enterprise/javaee.api/nbproject/project.properties [hunk 2]
```diff
@@ -20,9 +20,9 @@ release.external/javaee-api-6.0.jar=modules/ext/javaee-api-6.0.jar
 release.external/javaee-api-6.0-license.txt=modules/ext/javaee-api-6.0-license.txt
 release.external/javaee-web-api-6.0.jar=modules/ext/javaee-web-api-6.0.jar
 release.external/javaee-web-api-6.0-license.txt=modules/ext/javaee-web-api-6.0-license.txt
-release.external/javax.annotation.jar=modules/ext/javaee6-endorsed/javax.annotation.jar
-release.external/javax.annotation-license.txt=modules/ext/javaee6-endorsed/javax.annotation-license.txt
-release.external/jaxb-api-osgi.jar=modules/ext/javaee6-endorsed/jaxb-api-osgi.jar
-release.external/jaxb-api-osgi-license.txt=modules/ext/javaee6-endorsed/jaxb-api-osgi-license.txt
-release.external/webservices-api-osgi.jar=modules/ext/javaee6-endorsed/webservices-api-osgi.jar
-release.external/webservices-api-osgi-license.txt=modules/ext/javaee6-endorsed/webservices-api-osgi-license.txt
+release.external/javax.annotation-api-1.2.jar=modules/ext/javaee6-endorsed/javax.annotation-api-1.2.jar
```
[the remark this answers is not in the document]
Comment (rtaneja1) [reply enterprise/javaee.api/nbproject/project.properties]: Done, Thanks,

**Comment by geertjanw** [comment]
Great, once the above are done, let's merge.

**Comment by rtaneja1** [comment]
Thanks Matthias, included both changes.

**Pull request merged**